jetbrains-core/src/software/aws/toolkits/jetbrains/settings/UpdateLambdaSettings.kt

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.settings
+
+import com.intellij.openapi.components.PersistentStateComponent
+import com.intellij.openapi.components.ServiceManager
+import com.intellij.openapi.components.State
+import com.intellij.openapi.components.Storage
+
+@State(name = "updateLambdaSettings", storages = [Storage("aws.xml")])
+class UpdateLambdaSettings : PersistentStateComponent<UpdateSettings> {
+    private var state = UpdateSettings()
+
+    override fun getState(): UpdateSettings? = state
+    override fun loadState(state: UpdateSettings) {
+        this.state = state
+    }
+
+    fun useContainer(stackArn: String): Boolean? = state.updateConfigs[stackArn]?.useContainer
```

Comment:
`stackArn` is this copy-pasta?

Reply:
Sadly no, but will fix the name with the below suggestion as well